doc/user_guide/usage/run.rst

```diff
@@ -52,6 +52,13 @@ If the analyzed sources use implicit namespace packages (PEP 420), the source ro
 be specified using the ``--source-roots`` option. Otherwise, the package names are
 detected incorrectly, since implicit namespace packages don't contain an ``__init__.py``.
 
+With globbing pattern
+---------------------
+
+It is also possible to specify both directories and files using globbing patterns::
+
+   pylint [options] packages/*/src
+
```

Comment:
Imo this should be in the ``On module packages or directories`` paragraph

Reply:
Yet it's also applicable to `On files` - it can't be in both paragraphs at the same time

Reply:
IMO renaming the headline to `Globbing support` would be good, as it makes it a bit clearer that this section just informs you about the possibility to use glob patterns.

Reply:
Globbing permits to handle file/module/package, the distinction could be removed entirely maybe ?

Reply:
IMHO, that section of docs should be refactored, yet I don't have enough knowledge to do that confidently